src/test/impact-metrics/metric-types.test.ts

```diff
@@ -0,0 +1,68 @@
+import test from 'ava';
+import { ImpactMetricRegistry, MetricLabels } from '../../impact-metrics/metric-types';
+
+test('Counter increments by default value', (t) => {
+  const registry = new ImpactMetricRegistry();
+  const counter = registry.counter({ name: 'test_counter', help: 'testing' });
+
+  counter.inc();
+
+  const result = registry.collect();
+  const metric = result.find((m) => m.name === 'test_counter');
+
+  t.is(metric?.samples.length, 1);
```

Comment:
now sure about this testing library but would be nice to have assertions that verify the whole object that a person can easily parse